sharedb-codemirror.js

```diff
@@ -54,7 +54,7 @@ ShareDBCodeMirror.attachDocToCodeMirror = function(shareDoc, codeMirror, options
       shareDoc.on('op', shareDBOpListener);
     },
     onStop: function() {
-      shareDoc.removeListener('op', shareDBOpListener);
+      shareDoc.destroy();
```

Comment:
I'm not sure we want to fully destroy the doc here. In a larger application, ShareDBCodeMirror could be one of several components hooked up to ShareDB. I think it should be able to detach itself from the doc when it's no longer needed while leaving the doc in a usable state. For example, the user might navigate away from a page with a text editor to a settings page; the CodeMirror instance would be destroyed, but the doc would still be needed.

Instead of destroying the doc, is there some additional cleanup that we need to do in addition to removing the "op" listener?

Also, any new doc methods we use need to be added to `test/mocks/mock-doc.js` as well. It's a mocked-out version of the ShareDB doc that's used in the tests.

Reply:
I think it was my fault about this part. I turned the code back